das-dashboard/backend/services/query_services.py

```
@@ -0,0 +1,157 @@
+                "This proxy command does not exist/is not permitted in the current context."
+            )
+
+        return handlers[command_type](command_type, command_text)
+
+    def _get_query_parameters(self, command_type: str, command_text: str) -> Response:
+        return self._post_execution(command_type, command_text)
+
+    def _set_query_parameters(self, command_type: str, command_text: str) -> Response:
+        custom_parameters_dict = json.loads(command_text)
+        if not custom_parameters_dict:
+            raise CustomValueError("No query parameters were provided.")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)